banditpylib/data.proto

```diff
@@ -54,6 +54,18 @@ message Actions {
   repeated ArmPullsPair arm_pulls_pairs = 1;
 }
 
+// Next tag: 3
+message CollaborativeActions {
+  repeated ArmPullsPair arm_pulls_pairs = 1;
+
+  enum StateType {
```

Comment:
You can put `StateType` under `Actions` and add a comment `This is used by Collaborative learners`.